setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = strava_py
```

Comment:
This is the name that would be used for PyPI. Currently https://pypi.org/project/strava_py/ is unused (and https://pypi.org/project/strava/ is taken).